lib/src/editor/ui/editor.dart

```diff
@@ -129,31 +133,30 @@ class _DevicePreviewCanvas extends StatelessWidget {
 }
 
 class _Doc extends StatelessWidget {
-  List<ComponentState> states;
-  ComponentState? currentState;
+  final List<ComponentState> states;
+  final ComponentState? currentState;
+
   _Doc(this.states, this.currentState);
+
   @override
   Widget build(BuildContext context) {
-    return SingleChildScrollView(
-      child: Center(
-        child: Column(
-          children: [
-            ...states
-                .where((i) => i.parent == currentState?.parent)
-                .map(
-                  (item) => DocPanel(
-                    stateName: item.stateName,
-                    docs: item.docs,
-                    component: item.builder(
-                      context,
-                      context.watch<CanvasDelegateProvider>().storyProvider!,
-                    ),
-                  ),
-                )
-                .toList()
-          ],
-        ),
-      ),
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        ...states
+            .where((i) => i.parent == currentState?.parent)
+            .map(
+              (item) => DocPanel(
+                docName: item.docName,
+                component: item.builder(
+                  context,
+                  context.watch<CanvasDelegateProvider>().storyProvider!,
+                ),
+                stateName: item.stateName,
+              ),
+            )
+            .toList()
+      ],
```

Comment:
No need for the spread operator if it's the only thing in the list

```suggestion
      children: states
        .where((i) => i.parent == currentState?.parent)
        .map(
          (item) => DocPanel(
            docName: item.docName,
            component: item.builder(
              context,
              context.watch<CanvasDelegateProvider>().storyProvider!,
            ),
            stateName: item.stateName,
          ),
        )
        .toList(),
```